### PR TITLE
Update version with patch.

### PR DESCRIPTION
**Pull request by jvanderaa**
Sets patch release.

### CODE REVIEW

**Comment by jvanderaa** [comment]
Fixes #311

**Review by glennmatthews — COMMENTED** [review]
Although it's not spelled out yet in the project docs so far as I can see, I think the usual process for a release would be to do a release-branch PR into `main` (including release-note updates, etc.) rather than directly bumping the version in `develop`. Happy to collaborate with you on this if needed.

**Comment by jvanderaa** [comment]
So I would cut a new branch off of develop, PR that to main, then merge main back to develop?

**Comment by glennmatthews** [comment]
That's how we do things in core at least.

**Comment by jvanderaa** [comment]
I can do that then. What I've been doing on pynautobot, nautobot-ansible is:

merge pieces into develop
when ready PR develop into main and then cut release.

**Comment by chadell** [comment]
I guess the changelog is failing because the `release` file? is the first time I see it and I have not used it before for the releases

**Comment by glennmatthews** [comment]
I see that the target of the PR was changed correctly to `main`; in that case we need to run `towncrier` to get the *existing* change fragments collected into the release-note.

**Comment by jvanderaa** [comment]
Ok, this was a case of not following the new documentation. I have some questions on it with one change that we should make. The part about tags. It says to create a tag. Shouldn't that be `Create a Release`? What are the differences?

**Comment by jvanderaa** [comment]
@chadell I think this should be all set.

File: changes/311.release [hunk 1]
```diff
@@ -0,0 +1 @@
+Released 2.2.3.
```
Comment (glennmatthews) [inline changes/311.release:1]: This file shouldn't be needed for a PR into `main`. The documentation isn't as clear as it could be.

**Review by glennmatthews — APPROVED** [review]
The release process after this is merged should indeed be to create a `Release` and in the process of doing so, have GitHub create a `v2.2.3` `tag` for the release.

File: changes/311.release (hunk 1 above)
Comment (jvanderaa) [reply changes/311.release]: It was left over. Documentation (which I will tackle, you know me now :) ) was fine.

**Review by chadell — APPROVED** [review]
LGTM

**Pull request merged**